### DISCUSSION TITLE
[Feature] Allow Multiple Vault Endpoints

### DISCUSSION THREAD

**Discussion by Thundercraft5** [coddingtonbear/obsidian-web#148] [Ideas]
Allowing Obsidian Web to work with multiple endpoints at the same time can allow for very good and secure Vault collaboration over local networks/VPNs, even over separate networks (using ZeroTier).

**Idea**
Sending a request to the Vault(s) would work just the same, except they would be sent to multiple servers.

**Comment by coddingtonbear** [comment]
I'm afraid supporting this would require substantial amount of additional complexity in that it creates situations in which requests can be _partially_ OK if some subset of requests completes with different status codes from others.  Given that this is probably also a very niche feature, I'm afraid I'd need a lot, lot more convincing before being up for supporting such functionality even if the logic were written by somebody in the community.
